src/main/java/com/aws/greengrass/deployment/model/DeploymentTask.java

```diff
@@ -10,4 +10,7 @@
 public interface DeploymentTask extends Callable<DeploymentResult> {
     @Override
     DeploymentResult call() throws InterruptedException;
+
+    default void cancel() {
+    }
```

Comment:
Do we need this?

Reply:
Default Deployment also inherits from DeploymentTask.
We can either define a no-op default function here, for define it as no-op in Default Deployment